packages/fast-color-explorer/app/swatch.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import manageJss, { ComponentStyleSheet } from "@microsoft/fast-jss-manager-react";
+import { ComponentTypes } from "./state";
+import classnames from "classnames";
+import { elevation, ElevationMultiplier } from "@microsoft/fast-components-styles-msft";
+import { ColorsDesignSystem } from "./design-system";
+import { contrastRatio, parseColorHexRGB } from "@microsoft/fast-colors";
+
+export enum SwatchStates {
+    rest = "rest",
+    hover = "hover",
+    active = "active",
+    disabled = "disabled",
+    selected = "selected",
+}
+
+interface SwatchClassNameContract {
+    swatch: string;
+}
+
+interface SwatchManagedClasses {
+    managedClasses: SwatchClassNameContract;
+}
+
+interface SwatchProps extends SwatchManagedClasses {
+    /**
+     * The color of the swatch
+     */
+    color: string;
+
+    /**
+     * The background color that the swatch is relative to
+     */
+    backgroundColor: string;
+
+    /**
+     * The color of text against the swatch
+     */
+    textColor: string;
+
+    /**
+     * The type of component the swatch maps to
+     */
+    component?: ComponentTypes;
+
+    /**
+     * The UI state the swatch maps to
+     */
+    state?: SwatchStates;
+
+    /**
+     * Show contrast between color and background color
+     */
+    showBackgroundContrast?: boolean;
+
+    /**
+     * Show contrast between color and background color
+     */
+    showTextContrast?: boolean;
+}
+
+const styles: ComponentStyleSheet<SwatchClassNameContract, ColorsDesignSystem> = {
+    swatch: {
+        height: "36px",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        margin: "4px 0",
+        transition: "background-color .05s ease-in",
+        width: "180px",
+        fontSize: "12px",
+        borderRadius: "18px",
+        boxShadow: (elevation(ElevationMultiplier.e2, "#000")(undefined as any) as any)
```

Comment:
Can be applyElevation.

Reply:
I'll clean up in another PR - just trying to migrate.